src/lib/Runner.ts

```diff
@@ -65,6 +69,18 @@ export class Runner {
   };
 
   areMaxConcurrentBuildsRunning = async (processingLandRequestStatuses: LandRequestStatus[]) => {
+    if (processingLandRequestStatuses.length > MAX_LAND_DEPENDENCIES) {
+      // Set a hard limit on the total number of running + awaiting-merge + merge
+      Logger.warn('Hit max number of landkid dependencies', {
```

Comment:
Does this mean that the current request will just remain in the queue until the next "dependency slot" is free?